packages/angular_devkit/build_optimizer/src/build-optimizer/rollup-plugin.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * @fileoverview This adapts the buildOptimizer to run over each file as it is
+ * processed by Rollup. We must do this since buildOptimizer expects to see the
+ * ESModules in the input sources, and therefore cannot run on the rollup output
+ */
+
+import * as path from 'path';
+import { buildOptimizer } from './build-optimizer';
+import { RawSourceMap } from 'source-map';
+
+const DEBUG = false;
+
+export default function optimizer() {
+  return {
+    name: 'build-optimizer',
+    transform: (content: string, id: string): {code: string, map: RawSourceMap}|null => {
+      const isSideEffectFree = id.indexOf("packages/core/core.es5_esm") >= 0;
```

Comment:
Can you make this a config item for the plugin instead of a hardcoded path please?

Reply:
done